Makefile

```diff
@@ -1,3 +1,22 @@
+# Directory containing the content to validate. Default is "content".
+# It can be overridden by setting the CONTENT_DIR environment variable.
+# Example: CONTENT_DIR=content/manuals/compose make vale
+CONTENT_DIR := $(or $(CONTENT_DIR), content)
+
+# Docker image to use for vale.
+VALE_IMAGE := jdkato/vale:latest
+
+.PHONY: vale
+vale: ## run vale
+	docker run --rm -v $(PWD):/docs \
+		-w /docs \
+		-e PIP_BREAK_SYSTEM_PACKAGES=1 \
+		$(VALE_IMAGE) $(CONTENT_DIR)
+
+.PHONY: validate
+validate: ## run validations
+	docker buildx bake validate
```

Comment:
Not sure what the prupose of this change is but don't think that makes sense to have a make goal that just runs bake tbh. This would also not work on Windows host.

Reply:
You're right. I'll split the changes, and probably discard the validate one in a follow-up commit.

Regarding the vale goal, anything to highlight?